modules/css/loading-dots/README.md

```diff
@@ -1,12 +1,12 @@
-# Canvas Kit CSS Loading Animation
+# Canvas Kit CSS Loading Dots
 
-Components that display animations while a page or component loads.
+Component that displays animations while a page or component loads.
 
 <a href="../README.md">
   <img src="https://img.shields.io/badge/-maintenance mode-important" alt="Mainenance Mode" />
 </a>
 
-[> Workday Design Reference](https://design.workday.com/components/indicators/loading-animation)
+[> Workday Design Reference](https://canvas.workday.com/components/indicators/loading-dots)
```

Comment:
are we updating these links to be canvas.workday.com ? All but one component still uses design.workday.com

Reply:
I couldn't help it, but I can change it back. I noticed a recent change to bump this on that other component. I only changed it because I was editing this line.

Reply:
If I we do want to change it back, I'll want to go and change the other case where we use the updated link.

Reply:
I think we should be consistent in what url we are using - so if we are to go with canvas.workday.com we should do that everywhere and not just have one offs